pkg/csi/service/common/common_controller_helper.go

```diff
@@ -327,6 +327,11 @@ func IsVolumeSnapshotReady(ctx context.Context, client snapshotterClientSet.Inte
 			log.Warnf(msg)
 			return false, logger.LogNewErrorf(log, msg)
 		}
+		if svs == nil || svs.Status == nil || svs.Status.ReadyToUse == nil {
+			log.Warnf("Waiting for VolumeSnapshot %s/%s to be ready since %+vs, status field unset", namespace,
```

Comment:
Can you use similar language like this: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/2352/files#diff-126457def82d19c23f094cbdeca012cfb891da0e2fc7c3c02186d93e67f91a24R314

Reply:
Fixed